database/factories/ModelFactory.php

```diff
@@ -55,7 +55,7 @@
         'name' => $faker->company,
         'location' => $faker->postcode,
         'sponsor_id' => $sponsor_id, // a random sponsor
-        'payment_message' => "Placeholder Payment Message: $payment_sentence",
+        'payment_message' => "Thanks, your payment request has been accepted. $payment_sentence",
```

Comment:
This is a good candidate for the config/api.php message file. So we don't change in so many places.